org.adempiere.base/src/org/compiere/model/MTable.java

```diff
@@ -601,6 +601,22 @@ public PO getPO (ResultSet rs, String trxName)
 		return po;
 	}	//	getPO
 
+	/**
+	 * Get PO Class Instance
+	 * 
+	 * @param  uuID    UUID
+	 * @param  trxName transaction
+	 * @return         PO for Record or null
+	 */
+	public PO getPOByUU (String uuID, String trxName)
+	{
+		PO po = getPO(0, trxName);
+		if (po != null)

```

Comment:
don't think you can ever get po==null here.

Reply:
Agree